Source/UI/BillingForm/FramesFactory.swift

```diff
@@ -23,7 +23,7 @@ public enum FramesFactory {
          .phoneNumber(DefaultBillingFormPhoneNumberCellStyle())]
     }
 
-    static func getBillingFormViewController(style: BillingFormStyle?, data: BillingForm?, delegate: BillingFormViewModelDelegate?) -> UINavigationController? {
+    static func getBillingFormViewController(style: BillingFormStyle?, data: BillingForm?, delegate: BillingFormViewModelDelegate?, sender: UINavigationController?) -> UINavigationController? {
```

Comment:
```suggestion
    static func getBillingFormViewController(style: BillingFormStyle?, 
                                                                          data: BillingForm?, 
                                                                          delegate: BillingFormViewModelDelegate?, 
                                                                          presentingNavigationController: UINavigationController?) -> UINavigationController? {
```

Don't auto apply suggestion as indentation will probably trigger SwiftLint.